src/runtime/propertyobject.cs

```diff
@@ -31,7 +32,11 @@ public PropertyObject(PropertyInfo md)
         public static IntPtr tp_descr_get(IntPtr ds, IntPtr ob, IntPtr tp)
         {
             var self = (PropertyObject)GetManagedObject(ds);
-            MethodInfo getter = self.getter;
+            if (!self.info.Valid)
+            {
+                return Exceptions.RaiseTypeError(self.info.DeletedMessage);
+            }
+            MethodInfo getter = self.getter.Value;
```

Comment:
Should we check self.getter.Valid?

Reply:
No, but we should use UnsafeValue (doesn't throw), as null is a valid value in this context.

Reply:
No, but we should use UnsafeValue (doesn't throw), as null is a valid value in this context.